src/main/java/org/checkerframework/specimin/UnsolvedClass.java

```diff
@@ -121,7 +121,11 @@ public void addFields(String variableExpression) {
     this.classFields.add(variableExpression);
   }
 
-  /** This method sets the number of type variables for the current class */
+  /**
+   * This method sets the number of type variables for the current class
+   *
+   * @param numberOfTypeVariables number of type variable in this class.
+   */
```

Comment:
This change is harmless, but it wasn't necessary. I'm going to leave it in, but if we were in less of a hurry-up mode right now I'd insist you PR it separately.